//What is an error?

try {
    //let a=10
    console.log(a)
} catch (err) {
    console.log(err)
}


//What happens when error is occurred?
console.log("second")
console.log("third")
console.log("last")

//How to handle  error?
